<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoRecorderClass</class>
 <widget class="QMainWindow" name="VideoRecorderClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VideoRecorder</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="ImageView" name="imageView" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionEstimateFPS">
   <property name="text">
    <string>Estimate Capture FPS</string>
   </property>
  </action>
  <action name="actionIPCClientCount">
   <property name="text">
    <string>IPC Client: 0</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ImageView</class>
   <extends>QWidget</extends>
   <header>imageview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="videorecorder.qrc"/>
 </resources>
 <connections/>
</ui>
